### ISSUE TITLE
Jsonplaceholder website is down

### ISSUE THREAD

**Issue by andrasnemes** [typicode/jsonplaceholder#58]
Hello,

Navigating to https://jsonplaceholder.typicode.com/ results in an empty Heroku page with "There's nothing here, yet." printed on the page.

Thanks,
Andras

**Comment by hanjoosten** [comment]
Yeah. Hopefully there is someone able to get this awsome site up and running again. Thanks!

**Comment by typicode** [comment]
It's fixed now, sorry for the inconvenience and thanks @andrasnemes for the heads up!
I'll see how to prevent this from happening in the future.

**Comment by ikram0459** [comment]
this site ( https://jsonplaceholder.typicode.com/  ) is again down. @typicode  any update ?

**Comment by IrishChieftain** [comment]
https://jsonplaceholder.typicode.com/ has been down since yesterday and is giving 520 errors.

**Comment by flavius028** [comment]
[jsonplaceholder.typicode.com](url) has been down for a while, hopefully waiting for this awesome website up again :)

**Comment by typicode** [comment]
Seems to be working for me. Do you still have the issue? If so, what's the failing URL?

**Comment by flavius028** [comment]
> Seems to be working for me. Do you still have the issue? If so, what's the failing URL?
Its working now :) thank you for your reply! For while it was down :/
